misc/test/Gopkg.lock

```diff
@@ -3,26 +3,33 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:6978a38432a017763a148afbc7ce6491734b54292af7d3e969d84d2e9dd242e2"
```

Comment:
It looks like you are using a more recent `dep` than what we are using currently across Pulumi projects.  See https://github.com/pulumi/scripts/blob/master/ci/install-common-toolchain.sh for versions.  (I may have misled you in person yesterday pointing at https://github.com/pulumi/pulumi#development, which may pick up a newer version than we currently support).